api-platform-common-domain/src/test/scala/uk/gov/hmrc/apiplatform/modules/common/domain/services/InstantJsonFormatterSpec.scala

```diff
@@ -22,7 +22,7 @@ import play.api.libs.json._
 
 import uk.gov.hmrc.apiplatform.modules.common.utils.BaseJsonFormattersSpec
 
-class InstantJsonFormattersSpec extends BaseJsonFormattersSpec {
+class InstantJsonFormatterSpec extends BaseJsonFormattersSpec {
 
   "InstantJsonFormatters" should {
```

Comment:
Plural or not.  Your choice.

Reply:
Thanks - I fixed the test class's name to match the class under test, and missed this line. Will fix.